scripts/coverage.sh

```diff
@@ -32,12 +33,21 @@ generate_cover_data() {
 
     for pkg in "$@"; do
         f="$workdir/$(echo $pkg | tr / -).cover"
-        go test -v -covermode="$mode" -coverprofile="$f" "$pkg"
+        # go test -v -covermode="$mode" -coverprofile="$f" "$pkg"
+        # not using -v verbose flag, too much info to parse easily; output to file
+        go test -covermode="$mode" -coverprofile="$f" "$pkg" >> "$report"
     done
 
     echo "mode: $mode" >"$profile"
     grep -h -v "^mode:" "$workdir"/*.cover | $dir/exclude_ignore.py | egrep -v "$excludes" >>"$profile"
 }
 
-generate_cover_data $(go list ./... | grep -v /vendor/ | grep -v integration )
+generate_cover_data $(go list ./pkg/* | grep -v /vendor/ | grep -v integration )
 go tool cover -html="$profile" -o="$html"
```

Comment:
Just curious, where is the html page stored eventually?

Reply:
Not pretty sure, my guess that it will be eventually moved to /tmp/build/XXXXX/coverage/ for each CI job. But seems like we never use it.